Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -470,6 +470,28 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         message: { _ in .joinConditionsUsingComma },
         moveFixIt: { ReplaceTokensFixIt(replaceTokens: $0, replacement: trailingComma) }
       )
+    } else if let unexpected = node.unexpectedBetweenConditionAndTrailingComma,
+      let availability = node.condition.as(AvailabilityConditionSyntax.self),
+      let comparisonOperator = unexpected.oneTokenSatisfying(satisfying: { $0.tokenKind == .binaryOperator("==") }),
+      let falseKeyword = unexpected.oneTokenSatisfying(satisfying: { $0.tokenKind == .keyword(.false) })
```

Comment:
But this would also be `true` if `unexpected` is `false ==` (different order) or `== false some more garbage` right?

What do you think about adding a `twoTokens(first: (TokenSyntax) -> Bool, second: (TokenSyntax) -> Bool) -> Bool` function similar to `onlyToken` in SyntaxExtensions.swift?